### ISSUE TITLE
uikit-react-public accessibility

### ISSUE THREAD

**Issue by jhughes982** [ucl-arc-tre/portal#128]
- As I was working through #125 I noticed that the UCL uikit-react-public `Footer` component fails a 'moderate' accessibility test.
- At first glance, it looks like their footer component is not actually wrapped in a `<footer>` element and it is instead encapsulated by a `<div>`. Presumably this is not good for accessibility.
- This has got me wondering about the other components in the library and whether they follow html semantic and accessibility best practices.

# Definition of done
- [x] We open an issue on the uikit-react-public repo and raise the above issue
